detect_secrets/plugins/common/constants.py

```diff
@@ -6,11 +6,12 @@
     for r in [
         r'[ \t]+{} *pragma: ?whitelist[ -]secret{}[ \t]*$'.format(start, end)
         for start, end in (
-            ('#', ''),              # e.g. python or yaml
-            ('//', ''),             # e.g. golang
-            (r'/\*', r' *\*/'),     # e.g. c
-            ('\'', ''),             # e.g. visual basic .net
-            ('--', ''),             # e.g. sql
+            ('#', ''),                 # e.g. python or yaml
+            ('//', ''),                # e.g. golang
+            (r'/\*', r' *\*/'),        # e.g. c
+            ('\'', ''),                # e.g. visual basic .net
+            ('--', ''),                # e.g. sql
+            ('<!--[# \t]*', ' *-->'),  # e.g. xml
```

Comment:
Can you make this regex [lazy](https://stackoverflow.com/questions/2301285/what-do-lazy-and-greedy-mean-in-the-context-of-regular-expressions)  for performance reasons, I think `*` to `*?` in this case, and use the `r'` prefix just to denote that it is also a regex.

Reply:
@KevinHock sorry for the delay. The comment should be addressed now.

Reply:
No worries, looks great to me 👍